lib/srv/app/server.go

```diff
@@ -142,6 +146,13 @@ func (c *Config) CheckAndSetDefaults() error {
 		}
 		c.Cloud = cloud
 	}
+	if c.ConnectionLimiter == nil {
+		connectionLimiter, err := limiter.NewLimiter(limiter.Config{})
```

Comment:
What does empty config mean? No limits or default limits? Might be helpful to clarify with a comment.